2022-03-10 23:10:53,660 DEBUG decoration.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-10 23:10:53,660 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-10 23:10:53,660 DEBUG server.py Аргументы успешно загружены.
2022-03-10 23:10:53,660 DEBUG decoration.py Была вызвана функция arg_parser c параметрами ('8100', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-10 23:10:53,691 INFO core.py Запущен сервер, порт для подключений: 8100 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-10 23:11:01,468 INFO core.py Установлено соедение с ПК ('127.0.0.1', 58032)
2022-03-10 23:11:01,484 INFO core.py Установлено соедение с ПК ('127.0.0.1', 58033)
2022-03-10 23:11:01,999 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-10 23:11:01,999 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646943061.4840786, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-10 23:11:01,999 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-10 23:11:01,999 DEBUG core.py Correct username, starting passwd check.
2022-03-10 23:11:01,999 DEBUG core.py Auth message = {'response': 511, 'bin': '13e3c7dabf0feff0b0f97b917c7292271d5e83d3fa67daa1edea179f162d09efd9f9bd391c9ef3c5339a1f3f79c9bda16cd10b3834c40c5e5496874cba9cf38b'}
2022-03-10 23:11:01,999 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>, {'response': 511, 'bin': '13e3c7dabf0feff0b0f97b917c7292271d5e83d3fa67daa1edea179f162d09efd9f9bd391c9ef3c5339a1f3f79c9bda16cd10b3834c40c5e5496874cba9cf38b'}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2022-03-10 23:11:01,999 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2022-03-10 23:11:01,999 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>, {'response': 200}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2022-03-10 23:11:02,015 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-10 23:11:02,015 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646943061.4840786, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-10 23:11:02,015 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-10 23:11:02,015 DEBUG core.py Correct username, starting passwd check.
2022-03-10 23:11:02,015 DEBUG core.py Auth message = {'response': 511, 'bin': 'f17cded894355c8b4bc1aeba3afd54e708fb8c9431d45c0c6a992bcbbba1773bd02cbb16f01081696db15300cff15deaa4cecfce4b6ba589b11a0926ae1d4c5c'}
2022-03-10 23:11:02,015 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>, {'response': 511, 'bin': 'f17cded894355c8b4bc1aeba3afd54e708fb8c9431d45c0c6a992bcbbba1773bd02cbb16f01081696db15300cff15deaa4cecfce4b6ba589b11a0926ae1d4c5c'}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2022-03-10 23:11:02,029 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2022-03-10 23:11:02,031 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>, {'response': 200}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2022-03-10 23:11:02,547 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-10 23:11:02,547 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646943061.999703, 'account_name': 'test1'}
2022-03-10 23:11:02,547 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>, {'response': 202, 'data_list': ['test1', 'test2']}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2022-03-10 23:11:02,547 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-10 23:11:02,547 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646943062.0306516, 'account_name': 'test2'}
2022-03-10 23:11:02,547 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>, {'response': 202, 'data_list': ['test1', 'test2']}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2022-03-10 23:11:03,061 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-10 23:11:03,061 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646943062.5628035, 'user': 'test1'}
2022-03-10 23:11:03,061 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>, {'response': 202, 'data_list': ['test2']}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2022-03-10 23:11:03,061 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-10 23:11:03,061 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646943062.5628035, 'user': 'test2'}
2022-03-10 23:11:03,061 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>, {'response': 202, 'data_list': ['test1']}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2022-03-10 23:11:12,703 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-10 23:11:12,703 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1646943072.563201, 'account_name': 'test2'}
2022-03-10 23:11:12,703 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2022-03-10 23:11:14,726 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-10 23:11:14,726 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1646943074.359675, 'account_name': 'test1'}
2022-03-10 23:11:14,726 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2022-03-10 23:11:18,298 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-10 23:11:18,298 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'sender': 'test1', 'to': 'test2', 'time': 1646943078.0167816, 'mess_text': 's3YhmRcR22zcdZg14GTDWD5Dao8vGOKjpxmbTcIeT8z4Vmgo0QZhlZLtVlcTVj8TdaawbJCdeqVi9eGcalueD4gfBm21jmqX/EucfmdkuItg35qSPgZT8v9nhCUl89Oztn+CFioiTEXgGkKIVW7F51bOJuV9gxcooWZW7jCtyaV6k1HX9btrUpLZNdXT1vA5avIcc9jynPI5cR0KQQXZ2xs9YlRH5N1Xm1xhDhvCe1VghnACDvv0i46vgzuEHqYDPmGCvRA09wV0dO29Nb+C+qhiP4H7oqmJ3AHvNklQiNyqVCfet4Qgiha1Z382xJM7S5FmEaxhPzyAf8IbWken4g=='}
2022-03-10 23:11:18,311 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>, {'action': 'message', 'sender': 'test1', 'to': 'test2', 'time': 1646943078.0167816, 'mess_text': 's3YhmRcR22zcdZg14GTDWD5Dao8vGOKjpxmbTcIeT8z4Vmgo0QZhlZLtVlcTVj8TdaawbJCdeqVi9eGcalueD4gfBm21jmqX/EucfmdkuItg35qSPgZT8v9nhCUl89Oztn+CFioiTEXgGkKIVW7F51bOJuV9gxcooWZW7jCtyaV6k1HX9btrUpLZNdXT1vA5avIcc9jynPI5cR0KQQXZ2xs9YlRH5N1Xm1xhDhvCe1VghnACDvv0i46vgzuEHqYDPmGCvRA09wV0dO29Nb+C+qhiP4H7oqmJ3AHvNklQiNyqVCfet4Qgiha1Z382xJM7S5FmEaxhPzyAf8IbWken4g=='}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2022-03-10 23:11:18,311 INFO core.py Отправлено сообщение пользователю test2 от пользователя test1.
2022-03-10 23:11:18,313 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>, {'response': 200}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2022-03-10 23:11:20,864 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-10 23:11:20,864 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'sender': 'test2', 'to': 'test1', 'time': 1646943080.631525, 'mess_text': 'NvrmOA9FcrtWlVQWq6G7rgvjKygFbN2L5YgKPhAt4LWvhcas3IKcOvkwhmKg+rdTQ2+Oedsb9Ie6onnqEq9lvNWOy8x8BoghHdUpVVe5sysf7vdKmDWgjtXvmjFwya0lA+OkSQJPKWv7P5E8Wod5zUGJvO+jsO9JU+6FgpUZYclkD+qIQhz8kTAoBFl1Z38HDRKdlfcE2WxLy2kQmFwVNf9zMRFv/EOgp9g6KCikMGmb/aR5MkmJQe2MkhlYX4um2ni6zTK1nlGhWVVIxduI5NHr0EtEW/8CqNQHlyJa7x9ZwnNyo1W03P4FJmLGoed/nrDQFv5SRwYw+jqNI3Sd2w=='}
2022-03-10 23:11:20,873 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58032)>, {'action': 'message', 'sender': 'test2', 'to': 'test1', 'time': 1646943080.631525, 'mess_text': 'NvrmOA9FcrtWlVQWq6G7rgvjKygFbN2L5YgKPhAt4LWvhcas3IKcOvkwhmKg+rdTQ2+Oedsb9Ie6onnqEq9lvNWOy8x8BoghHdUpVVe5sysf7vdKmDWgjtXvmjFwya0lA+OkSQJPKWv7P5E8Wod5zUGJvO+jsO9JU+6FgpUZYclkD+qIQhz8kTAoBFl1Z38HDRKdlfcE2WxLy2kQmFwVNf9zMRFv/EOgp9g6KCikMGmb/aR5MkmJQe2MkhlYX4um2ni6zTK1nlGhWVVIxduI5NHr0EtEW/8CqNQHlyJa7x9ZwnNyo1W03P4FJmLGoed/nrDQFv5SRwYw+jqNI3Sd2w=='}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2022-03-10 23:11:20,873 INFO core.py Отправлено сообщение пользователю test1 от пользователя test2.
2022-03-10 23:11:20,875 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1340, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 58033)>, {'response': 200}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2022-03-11 20:23:40,868 DEBUG decoration.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 20:23:40,868 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['C:/Python_GK/PyQT/Lesson_2/server.py']
2022-03-11 20:23:40,870 DEBUG server.py Аргументы успешно загружены.
2022-03-11 20:23:40,871 DEBUG decoration.py Была вызвана функция arg_parser c параметрами ('8100', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 20:23:40,921 INFO core.py Запущен сервер, порт для подключений: 8100 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-11 20:24:22,311 DEBUG decoration.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции <module>
2022-03-11 20:27:56,456 DEBUG decoration.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 20:27:56,457 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-11 20:27:56,458 DEBUG server.py Аргументы успешно загружены.
2022-03-11 20:27:56,459 DEBUG decoration.py Была вызвана функция arg_parser c параметрами ('8100', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 20:27:56,485 INFO core.py Запущен сервер, порт для подключений: 8100 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-11 20:28:05,727 INFO core.py Установлено соедение с ПК ('127.0.0.1', 55709)
2022-03-11 20:28:05,750 INFO core.py Установлено соедение с ПК ('127.0.0.1', 55710)
2022-03-11 20:28:05,758 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1328, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 55709)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 20:28:05,758 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647019685.7339294, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 20:28:05,762 INFO core.py Клиент ('127.0.0.1', 55709) отключился от сервера.
2022-03-11 20:28:06,277 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 55710)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 20:28:06,277 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647019685.7579248, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 20:28:06,279 INFO core.py Клиент ('127.0.0.1', 55710) отключился от сервера.
2022-03-11 20:29:44,159 INFO core.py Установлено соедение с ПК ('127.0.0.1', 55717)
2022-03-11 20:29:44,672 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=972, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 55717)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 20:29:44,673 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647019784.1657372, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 20:29:44,674 INFO core.py Клиент ('127.0.0.1', 55717) отключился от сервера.
2022-03-11 20:30:30,688 INFO core.py Установлено соедение с ПК ('127.0.0.1', 55723)
2022-03-11 20:30:31,189 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 55723)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 20:30:31,190 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647019830.6950004, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 20:30:31,191 INFO core.py Клиент ('127.0.0.1', 55723) отключился от сервера.
2022-03-11 20:58:46,108 DEBUG decoration.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 20:58:46,108 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-11 20:58:46,110 DEBUG server.py Аргументы успешно загружены.
2022-03-11 20:58:46,111 DEBUG decoration.py Была вызвана функция arg_parser c параметрами ('8100', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 20:58:46,134 INFO core.py Запущен сервер, порт для подключений: 8100 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-11 20:58:54,581 INFO core.py Установлено соедение с ПК ('127.0.0.1', 55908)
2022-03-11 20:58:54,589 INFO core.py Установлено соедение с ПК ('127.0.0.1', 55909)
2022-03-11 20:58:54,591 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1184, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 55908)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 20:58:54,591 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647021534.588069, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 20:58:54,595 INFO core.py Клиент ('127.0.0.1', 55908) отключился от сервера.
2022-03-11 20:58:55,105 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 55909)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 20:58:55,105 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647021534.596069, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 20:58:55,106 INFO core.py Клиент ('127.0.0.1', 55909) отключился от сервера.
2022-03-11 20:59:35,590 INFO core.py Установлено соедение с ПК ('127.0.0.1', 55915)
2022-03-11 20:59:36,094 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 55915)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 20:59:36,094 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647021575.5966501, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 20:59:36,096 INFO core.py Клиент ('127.0.0.1', 55915) отключился от сервера.
2022-03-11 21:20:33,703 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56110)
2022-03-11 21:20:34,209 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56110)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 21:20:34,210 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647022833.7102373, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 21:20:34,211 INFO core.py Клиент ('127.0.0.1', 56110) отключился от сервера.
2022-03-11 21:24:42,274 DEBUG decoration.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 21:24:42,274 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-11 21:24:42,276 DEBUG server.py Аргументы успешно загружены.
2022-03-11 21:24:42,277 DEBUG decoration.py Была вызвана функция arg_parser c параметрами ('8100', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 21:24:42,301 INFO core.py Запущен сервер, порт для подключений: 8100 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-11 21:24:51,985 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56136)
2022-03-11 21:24:51,994 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56137)
2022-03-11 21:24:51,995 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56136)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 21:24:51,996 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647023091.9929624, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 21:24:52,000 INFO core.py Клиент ('127.0.0.1', 56136) отключился от сервера.
2022-03-11 21:24:52,503 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1328, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56137)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 21:24:52,504 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647023092.0019643, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 21:24:52,505 INFO core.py Клиент ('127.0.0.1', 56137) отключился от сервера.
2022-03-11 21:56:07,746 DEBUG decoration.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 21:56:07,746 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-11 21:56:07,748 DEBUG server.py Аргументы успешно загружены.
2022-03-11 21:56:07,748 DEBUG decoration.py Была вызвана функция arg_parser c параметрами ('8100', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 21:56:07,775 INFO core.py Запущен сервер, порт для подключений: 8100 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-11 21:56:18,650 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56373)
2022-03-11 21:56:18,655 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56374)
2022-03-11 21:56:19,163 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1032, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56373)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 21:56:19,163 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647024978.6572418, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 21:56:19,167 INFO core.py Клиент ('127.0.0.1', 56373) отключился от сервера.
2022-03-11 21:56:19,168 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56374)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 21:56:19,168 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647024978.6622405, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 21:56:19,169 INFO core.py Клиент ('127.0.0.1', 56374) отключился от сервера.
2022-03-11 22:03:17,327 DEBUG decoration.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 22:03:17,327 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-11 22:03:17,329 DEBUG server.py Аргументы успешно загружены.
2022-03-11 22:03:17,330 DEBUG decoration.py Была вызвана функция arg_parser c параметрами ('8100', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 22:03:17,357 INFO core.py Запущен сервер, порт для подключений: 8100 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-11 22:03:27,408 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56442)
2022-03-11 22:03:27,418 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56443)
2022-03-11 22:03:27,420 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56442)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 22:03:27,420 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647025407.4140656, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 22:03:27,424 INFO core.py Клиент ('127.0.0.1', 56442) отключился от сервера.
2022-03-11 22:03:27,936 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1328, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56443)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 22:03:27,937 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647025407.4254906, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 22:03:27,938 INFO core.py Клиент ('127.0.0.1', 56443) отключился от сервера.
2022-03-11 22:11:18,192 DEBUG decoration.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 22:11:18,192 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-11 22:11:18,194 DEBUG server.py Аргументы успешно загружены.
2022-03-11 22:11:18,194 DEBUG decoration.py Была вызвана функция arg_parser c параметрами ('8100', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 22:11:18,220 INFO core.py Запущен сервер, порт для подключений: 8100 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-11 22:11:26,608 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56591)
2022-03-11 22:11:26,621 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56592)
2022-03-11 22:11:26,622 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1332, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56591)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 22:11:26,623 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647025886.6148596, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 22:11:26,628 INFO core.py Клиент ('127.0.0.1', 56591) отключился от сервера.
2022-03-11 22:11:27,136 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1328, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56592)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 22:11:27,137 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647025886.6282003, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 22:11:27,138 INFO core.py Клиент ('127.0.0.1', 56592) отключился от сервера.
2022-03-11 22:12:10,125 DEBUG decoration.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 22:12:10,125 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-11 22:12:10,126 DEBUG server.py Аргументы успешно загружены.
2022-03-11 22:12:10,127 DEBUG decoration.py Была вызвана функция arg_parser c параметрами ('8100', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 22:12:10,200 INFO core.py Запущен сервер, порт для подключений: 8100 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-11 22:12:19,806 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56596)
2022-03-11 22:12:19,814 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56597)
2022-03-11 22:12:19,815 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1384, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56596)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 22:12:19,815 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647025939.8123639, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 22:12:19,819 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1384, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56596)>, {'response': 400, 'error': 'Пользователь не зарегистрирован.'}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2022-03-11 22:12:20,333 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=968, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56597)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 22:12:20,333 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647025939.8203623, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 22:12:20,335 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=968, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56597)>, {'response': 400, 'error': 'Пользователь не зарегистрирован.'}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2022-03-11 22:13:04,750 DEBUG decoration.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 22:13:04,750 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-11 22:13:04,752 DEBUG server.py Аргументы успешно загружены.
2022-03-11 22:13:04,752 DEBUG decoration.py Была вызвана функция arg_parser c параметрами ('8100', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-11 22:13:04,781 INFO core.py Запущен сервер, порт для подключений: 8100 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-11 22:13:14,002 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56611)
2022-03-11 22:13:14,007 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56612)
2022-03-11 22:13:14,518 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56611)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 22:13:14,518 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647025994.0096517, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 22:13:14,522 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56611)>, {'response': 400, 'error': 'Пользователь не зарегистрирован.'}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2022-03-11 22:13:14,523 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1124, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56612)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 22:13:14,523 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647025994.0136526, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 22:13:14,525 DEBUG decoration.py Была вызвана функция send_message c параметрами (<socket.socket fd=1124, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56612)>, {'response': 400, 'error': 'Пользователь не зарегистрирован.'}), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2022-03-11 22:14:03,853 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56634)
2022-03-11 22:14:03,861 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56635)
2022-03-11 22:14:03,862 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1772, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56634)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 22:14:03,862 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647026043.8594186, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwoAHA0Km34TGyZaxiyez\ncDYYtF37N3AVEFWyumKMOwEBgu06IeSDeffNOho6bhkunLMYCnT6hbp8Zx4chu6D\nGCxlZ8Bhg0U0GGIqFwsfHv4EnEWXoVx3j+J27s4sZEiONCp2gwi3kh5HAJVCjBEi\nh4vGsyn+7RWCvgcIrPD86ir+DJPyPGwd/ydg++On7TX3QW3jhLjtkbUEKg9liblU\ngz7Rp04s2iK5lQTPB4oH4phNMljWqgeQdCkKoCpQJYtbQl45jcmdDY9NdDBI3AbR\ng/GkgD+X7mjc4Y6EXjM/9TDzZ/3k5EkMnF5XeG3+hfsaXChcyGA0tRTsnbgLfbl5\nMQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 22:14:03,864 INFO core.py Клиент ('127.0.0.1', 56634) отключился от сервера.
2022-03-11 22:14:04,373 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1488, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56635)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 22:14:04,373 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647026043.8674185, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 22:14:04,375 INFO core.py Клиент ('127.0.0.1', 56635) отключился от сервера.
2022-03-11 22:14:24,343 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56637)
2022-03-11 22:14:24,850 DEBUG decoration.py Была вызвана функция get_message c параметрами (<socket.socket fd=1488, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8100), raddr=('127.0.0.1', 56637)>,), {}. Вызов из модуля common.utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2022-03-11 22:14:24,850 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647026064.3491535, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApB97Q7zHIUXdAUKxxc1V\nbPHSdhQJP58cfZ05GeSc8EI+BCsKJt0DyWeox5JBiSpa8idpZExtODyhtOCzCiNw\nvmpRLPTl7GKleAWb59QIDPQ7mhPrxLuqAWc81Wqtfd4b9xogYrjShC+8bUeO8/3J\n0Y00dMD0jwCMw6SKO8kSgqoa2W8JFErQ2/ab2ja72MUs5NSxMGMpPEzvw0eTjdce\ncJyxHaPiR9FyaglI4OuGlcdK7jERgJOCxdBVUaKs5iAIdh6wUn7NjMucRFIrXofz\nCrQ5/s7bEJ8ZpU4io4R/sYKnFumZ+2meXd7nmptIJQLYdUbQIXyKrF/Ibl7GE0Lr\n0QIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-11 22:14:24,852 INFO core.py Клиент ('127.0.0.1', 56637) отключился от сервера.
